### ISSUE TITLE
Make separate data models for "state" and "parameters"

### ISSUE THREAD

**Issue by WardLT** [ROVI-org/auto-soh#6]
Our current `SystemModel` requires users to annotate which attributes belong to different categories, such as "state" or "health" parameters. It would be cleaner to have those separated into two subclasses.

Current:

```python

class MySystemModel(SystemModel):
      attr_1
      attr_2
      state_params = ('attr_1',)
```

Proposed

```python

class State:
    attr_1

class Health:
    attr_2

class JointState():
    state = State()
    att_2 = Health()
```

**Comment by victorventuri** [comment]
Agreed, will look into starting that process early next week!

**Comment by WardLT** [comment]
Fixed by #11
